CONTRIBUTING.md

```diff
@@ -296,6 +296,113 @@ OS and is built by Android NDK. This often introduces compatibility issues as
 Android (specifically Termux) is not a standard platform. Do not expect there
 are exist package recipes available out-of-box.
 
+## Commit guidelines
+
+A commit message should look something like:
+```
+<commitType>(<repo>/<package>): (The change made/Short description of the change)
+
+[An optional but **highly recommended** commit message describing the changes made in the commit]
+
+[Fixes (termux/repo)#<issue number>]
```

Comment:
is this pattern "recognised" by github's web UI? 

Tests:

Just pure link: https://github.com/termux/termux-packages/issues/7834
With parentheses: (termux/termux-packages)#7834

just providing a link like https://github.com/termux/termux-packages/pull/10748 is quite handy

Reply:
Without parentheses works, so I suggest we use that

Reply:
Well, parentheses was intended to show that it is optional. That is, when referencing issue/PR from the same repository, the details inside parentheses would not be needed. But when referencing issues from other repositories, it's needed. It seems like this is misleading.

What do you think of something like:
```
Fixes #<issue number>
Fixes termux/<repo>#<issue number>
```

I guess this will not be mistakened